migrations/mystery_items.js

```diff
@@ -27,6 +27,8 @@ if (_id) {
   db.users.update({_id:_id}, update);
 } else {
   // multiple (once @ start of event)
-  db.users.update({'purchased.plan.customerId':{$ne:null}}, update, {multi:true});
+  db.users.find({'purchased.plan.dateTerminated':{$exists:true}}).forEach(function(user) {
```

Comment:
@kylestratis You're right regarding `findAndModify`. Any progress on @Alys's instructions?